.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ commands:
             - .cargo
 jobs:
 
-  rust-stable:
+  rust-stable-integration:
```

Comment:
Explain: Since the stable and nightly only covers the integration tests beyond other tests, so we cut the workload here.